xds/server_test.go

```diff
@@ -86,6 +86,10 @@ func (f *fakeGRPCServer) GracefulStop() {
 	f.gracefulStopCh.Send(nil)
 }
 
+func (f *fakeGRPCServer) GetServiceInfo() map[string]grpc.ServiceInfo {
+	return map[string]grpc.ServiceInfo{}
```

Comment:
This should either return `nil` or `panic("implement me")`.
We currently have no tests using this and when they need to use it, they would also need to provide a way by which the test can inject a `ServiceInfo` to be returned. So, returning nil or panicking sounds better to me at this point in time.

Reply:
Changed it to panic